### ISSUE TITLE
Steam ignores articles in game titles

### ISSUE THREAD

**Issue by ViBE-HU** [ValveSoftware/steam-for-linux#4514]
#### Your system information

* Steam client version: Actual latest beta
* Distribution (e.g. Ubuntu): Ubuntu 16.04 (but all platforms affected long time ago)
* Opted into Steam client beta?: [Yes] 
* Have you checked for system updates?: [Yes]

#### Please describe your issue in as much detail as possible:
A Story About My Uncle, The Floor is Jelly, The Silent Age, The Swapper, just a few titles what listed in incorrect order because Steam ignores articles. Please fix this.

**Comment by Tele42** [comment]
In my opinion, ignoring articles in titles is better. This practice is rather common when it comes to large sets of titles.

**Comment by ViBE-HU** [comment]
That's why favourites and categories created i guess. And it's pretty make no sense how big is a brand if you have a bunch of games and your loved games.

**Comment by gdrewb-valve** [comment]
This is deliberate and I don't believe it's going to change so I'm going to go ahead close this.

**Comment by ViBE-HU** [comment]
But why is it deliberate? What's the point of this? What's the point of categories and favourites then?

**Comment by gdrewb-valve** [comment]
Ignoring articles when alphabetizing names of books/movies/etc. is standard practice as otherwise you get large clumps of things under A, An and The.  For example check the tip in http://www.wikihow.com/Alphabetize:
`Ignore articles at the beginning of titles. You can disregard the words “a”, “an” or “the” if they begin a title as they are very common and can make searching through alphabetized information confusing.`
I don't understand your question about categories and favorites as that seems separate from the issue you have created which only refers to leading articles in lists.

**Comment by ViBE-HU** [comment]
I didn't know that. So it's a kind of English(?) standardisation. In my language it isn't a standard practice and that's why I don't find this logical. Titles usually start with articles but we don't ignore them here.

I though maybe Tele42 is right and it could be a reason. That's why I asked.

**Comment by gdrewb-valve** [comment]
I believe it's done in languages other than English but I don't know of any list.  I can see how it'd be surprising if you weren't used to it.  I'll reopen this as a feature request for an option to turn it on or off (or possibly an automatic language-based choice but I don't know if that's reasonable).

**Comment by ViBE-HU** [comment]
Would be good as a(n) opt-in/default. I would really appreciate that. It just fits better for some languages I guess. Thanks.

**Comment by stqn1** [comment]
I agree that it’s annoying. At first I was looking for the games at their correct place, and didn’t find them. Now I know so I have to mentally remember to remove the leading article, then think about the starting letter of the second word, then look there…

And I know that libraries do that (in France) but it doesn’t change the fact that it’s counter-intuitive for me. I think for in physical libraries it makes it easier to store books, but in a digital library there is no point doing that.

**Comment by ViBE-HU** [comment]
Same logic here. "Working" with correct information in correct order is much comfortable and that's how my mind sort the things.
